README.md

```diff
@@ -52,6 +98,7 @@ Allow user to create test database
 * Run `python manage.py migrate`
 * Run `python manage.py runserver localhost:8081`
 * The project is now running at [localhost:8081](http://localhost:8081)
+* [When Keycloak has been started](#original-keycloak), it is running at [localhost:8180](http://localhost:8180) and at [notification-service-keycloak:8180](http://notification-service-keycloak:8180) (if hostname is set)
```

Comment:
IMHO using [localhost:8180](http://localhost:8180) is detrimental, because it makes it possible for the link name and real URL to go out-of-sync. Just using http://localhost:8180 and letting the .md format reader understand that this is an URL and format it accordingly makes the out-of-sync problem go away.

Reply:
Fair point. Changed those.